src/components/layout/Navbar.tsx

```diff
@@ -42,29 +61,50 @@ const navItems: NavItem[] = [
     label: 'Verification',
     url: '/verification',
   },
+  {
+    label: 'Settings',
+    url: '/settings',
+  },
   {
     label: 'SearchSECO',
     pages: [
-      { label: 'Query', url: '/query' },
-      { label: 'Mining', url: '/mining' },
+      {
+        label: 'Query',
+        url: '/query',
+        icon: HiOutlineDocumentMagnifyingGlass,
+        description: `Query the SearchSECO database using your ${TOKENS.secoin.symbol} tokens.}`,
```

Comment:
There's a period at the end of this description, while there isn't one in the description of Mining page.
You know what they say, it's all in the details